DESCRIPTION

```diff
@@ -89,7 +90,9 @@ Suggests:
     ggplot2,
     reactlog (>= 1.0.0),
     magrittr,
-    yaml
+    yaml,
+    future,
```

Comment:
These packages aren't needed anymore, right?

Reply:
They're still used in `test-test-module`, actually.